presto-main/src/main/java/com/facebook/presto/execution/Output.java

```diff
@@ -29,16 +31,19 @@
     private final ConnectorId connectorId;
     private final String schema;
     private final String table;
+    private List<DateTime> lastDataCommitTimes;
 
     @JsonCreator
     public Output(
             @JsonProperty("connectorId") ConnectorId connectorId,
             @JsonProperty("schema") String schema,
-            @JsonProperty("table") String table)
+            @JsonProperty("table") String table,
+            @JsonProperty("lastDataCommitTimes") List<DateTime> lastDataCommitTimes)
```

Comment:
commit time is a hive connector specific concept. The main engine should have no idea what it is. So
- we should call it `ConnectorCommitOutput`
- The data type should either be `Object` with json serializer or `String` if we expect to output it in plain string format.

Reply:
Hi @highker, now I am trying to attach the entire connector commit handle to this object, which is json serailizable, how about that?

Reply:
@highker, attaching the entire ConnectorCommitHandle seems having a Json deserialization problem, since it is an abstract type. Errors are as follows. Ok, then I will try to use ConnectorCommitOutput instead.

Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `com.facebook.presto.spi.connector.ConnectorCommitHandle` (no Creators, like default construct, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
 at [Source: (String)"{
  "connectorId" : "connectorId",
  "schema" : "schema",
  "table" : "table",
  "connectorCommitHandle" : "INSTANCE"
}"; line: 5, column: 29] (through reference chain.

Reply:
Commented here: https://github.com/prestodb/presto/pull/17550#discussion_r843124138